kmxwasm/src/kmxwasm/tools.py

```diff
@@ -110,7 +110,9 @@ def explorer(self) -> KCFGExplore:
                     # port=39425,
                 )
         if not self.__kore_client:
-            self.__kore_client = KoreClient('localhost', self.__kore_server.port, bug_report=self.__bug_report)
+            self.__kore_client = KoreClient(
+                'localhost', self.__kore_server.port, bug_report=self.__bug_report, bug_report_id='kasmer'
```

Comment:
The logic for `bug_report_id` and `bug_report` requires both to be either None or not at the same time... :thinking: 
```suggestion
                'localhost',
                self.__kore_server.port,
                bug_report=self.__bug_report,
                bug_report_id='kasmer' if self.bug_report is not None else None,
```
(I checked that this parses the formatting check... )